### ISSUE TITLE
There might be a mistake in the lesson

### ISSUE THREAD

**Issue by Oregu404** [freeCodeCamp/freeCodeCamp#57976]
Step 54 “Responsive web design certification” in “Learn accessibility by building a quiz”. The question in this step is: “Give the .formrow elements a top margin, and some left and right padding. The rest of the padding values ​​should be 0.” - But we haven’t created a .formrow class before, and DOOM doesn’t have such an element at the moment.

https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-accessibility-by-building-a-quiz/step-54

**Comment by ilenia-magoni** [comment]
there is a `.formrow` element on line 90 of the html file, it was created in step 35

**Comment by Oregu404** [comment]
Thanks. Strange, I looked before and didn't see it and my browser didn't find it through the search on the page
